frontend/src/index.css

```
@@ -20,54 +18,328 @@
+    color: #ededed;
   }
 
   .btn-danger {
-    @apply inline-flex items-center justify-center gap-2 bg-red-500 text-white px-4 py-2.5 rounded-lg font-medium text-sm hover:bg-red-600 active:scale-[0.98] transition;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    background: transparent;
+    color: #ee0000;
+    border: 1px solid rgba(238, 0, 0, 0.35);
```

Stylelint (17.14.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)